### ISSUE TITLE
Web profiler error (CoreBundle depends on ShopBundle)

### ISSUE THREAD

**Issue by arjanfrans** [Sylius/ShopApiPlugin#677]
The web profiler breaks after I replaced the ShopBundle with the ShopApiBundle.

```
There are no registered paths for namespace "SyliusShop" in @SyliusCore/Collector/cart.html.twig at line 5.
```

It seems that the CoreBundle depends on a template from the ShopBundle.

**Comment by mamazu** [comment]
At least for the error templates yes.

**Comment by arjanfrans** [comment]
As a workaround I manually configured the `ShopBundle` namespace in the twig config:

```
twig:
    paths:
        '%kernel.project_dir%/templates': ~
        '%kernel.project_dir%/vendor/sylius/sylius/src/Sylius/Bundle/ShopBundle/Resources/views': 'SyliusShop'
```

**Comment by mamazu** [comment]
And the define your own templates?

**Comment by arjanfrans** [comment]
No, I didn't define my own templates. Symfony doesn't automatically fin the `SyliusShop` twig namespace, because the `ShopBundle` is not active. However, it is still installed as a part of the core installation. So I just manually configure the namespace here.

**Comment by mamazu** [comment]
Ah okay then I will add this to the documentation as a step during installation.

**Comment by arjanfrans** [comment]
@mamazu Thanks!

The better solution would be to fix this in the core bundle, I created an issue there. See https://github.com/Sylius/Sylius/issues/11918

**Comment by mamazu** [comment]
Thanks for your help.

**Comment by lchrusciel** [comment]
Hey Arjan,

thanks for reporting. We should fix it in the core library 👍

**Comment by lchrusciel** [comment]
Fixed already in #11958. Will be published with the next patch release
